docs/how-to-use-the-meeting-app-solution/README.md

```diff
@@ -0,0 +1,174 @@
+    - Key Length (optional).
+- RTMP Protocol: 
+    - Injection URL (only for `Pull` mode).
+    - Enable SSL (to stream over RTMP or RTMPS).
+
+![Call details view injection start setings](images/call-details-view-start-injection-settings.png)
+
+>**NOTE**: The injection URL for SRT in `Caller` mode or RTMP in `Pull` mode, is the URL where the Bot Service will establish the connection to pull the content.
+
+In the new view displayed please configure the settings and press the `Start` button 
+
+Finally, click on the `Start` button to start the injection. You will be returned to the call details view. Additionally, you can find the information related to the injection started by clicking on the `...` button in the `Card` of the [Injection Stream](#injection-stream) section.
+
+![Call details view injection expanded view](images/call-details-view-injection-expanded-view.png)
+
+>**NOTE**: You can mute/unmute the audio injected from the bot by clicking on the `Mic` button.
+
+>**NOTE**: To close the expanded view, please click on the avatar image located at the top right side.
+
+To stop the stream injection you just need to click on the same `Record` button in collapsed view or click on the `Stop` button in the expanded view.
+
+### How to extract media content from the meeting
+To extract media content from the meeting you need to go to the Main Streams or Participants section. In the corresponding stream `Card` you want to extract click on the `Record` button.
+
+![Call details view extraction start](images/call-details-view-start-extraction.png)
+
+Similar to the injection, a new view will be displayed with the options to configure and start the extraction. After that, you will be returned to the call details view. 
+
+![Call details view extraction start settings](images/call-details-view-start-extraction-settings.png)
+
+>**NOTE**: Based on the protocol configured and the mode selected you need to configure different options in the settings view.
+
+Finally, to start the extraction, click on the `Start` button located at the bottom right of the settings view.
+
+For information related to the extraction, go to the [Broadcast Outputs](#broadcast-outputs) section and click on the three dots `...` button at the right of the corresponding extraction `Card` to expand it.
+
+![Call details view extraction expanded view](images/call-details-view-extraction-expanded-view.png)
+
+>**NOTE**: To close the expanded view, please click on the avatar image located at the top right side. 
+
+### Stream key for RTMP protocol
+When the bot is joined into the meeting, a stream key is generated. This value is used for both, injection in push mode and extraction in pull mode. 
+
+The stream key is displayed (obfuscated) in the settings view. It can be regenerated by clicking on the `Refresh Stream Key` button placed at the right where the value is displayed.
+
+![Call details view stream key](images/call-details-view-stream-key.png)
```

Comment:
**issue (blocking)**: Looks like this image is missing.

Reply:
Updated